pkg/grafana/synthetic-monitoring.go

```diff
@@ -71,22 +71,24 @@ func getRemoteCheck(uid string) (*Check, error) {
 		return nil, err
 	}
 	for _, check := range checks {
-		if check.UID() == uid {
+		if check.Job() == uid {
```

Comment:
In some places I see UID replace by `Name` and here it is by `Job`, is that intended?

Reply:
Yeah, this is an internal thing to SM, I suspect this code could be simplified though, potentially loosing the `Check` struct.